setup.py

```diff
@@ -22,6 +25,13 @@
 
 # Allow manually disabling seccomp on old kernels. WSL doesn't have seccomp.
 has_seccomp = sys.platform.startswith('linux') and not is_wsl and os.environ.get('DMOJ_USE_SECCOMP') != 'no'
+try:
+    parallel = int(os.environ['DMOJ_PARALLEL'])
+except (KeyError, ValueError):
+    parallel = cpu_count()
```

Comment:
This can fail with `NotImplementedError`, should we provide a default value of 1 if it does so?

Reply:
Good catch.